subliminal/providers/addic7ed.py

```diff
@@ -108,7 +82,7 @@ def get(self, url, params=None):
 
         """
         try:
-            r = self.session.get(self.server + url, params=params, timeout=10)
+            r = self.session.get(self.server + url, params=params, timeout=30)
```

Comment:
I think 10s timeout is plenty, if the provider doesn't respond by then, they are experiencing serious trouble.